### PR TITLE
Download specific versions of plugins

### PR DESCRIPTION
**Pull request by owu-1**
Download specific versions of plugins so I know what versions the plugins are on. Without knowing that its hard to know if errors happening are because of older versions. Also if I'm reporting errors I need to know the version. Some plugins don't show their build number in their version command output.

- [ ] ~~Don't download versions we already have~~

### CODE REVIEW

File: mods.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 mkdir -p plugins
 cd plugins || exit
```
Comment (y2k04) [inline mods.sh:7]: Why is the exit command piped from a change directory command? Was that just for debugging and you forgot to remove it, or what?

Comment (owu-1) [reply mods.sh:7]: Its like if the cd fails then exit the script

Comment (owu-1) [reply mods.sh:7]: I'm linting it with this https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck
And heres its explaination https://www.shellcheck.net/wiki/SC2164

Comment (y2k04) [reply mods.sh:7]: Cool

**Review by owu-1 — COMMENTED** [review]
Linter doesnt like it without the './' : https://www.shellcheck.net/wiki/SC2035

**Comment by y2k04** [comment]
> Linter doesnt like it without the './' : https://www.shellcheck.net/wiki/SC2035

Ok

**Pull request merged**